internal/ethapi/api.go

```diff
@@ -1335,11 +1335,14 @@ type RPCTransaction struct {
 	// L1 message transaction fields:
 	Sender     common.Address  `json:"sender,omitempty"`
 	QueueIndex *hexutil.Uint64 `json:"queueIndex,omitempty"`
+
+	// wrapped traces, currently only available for `scroll_getSkippedTransaction` API, when `MinerStoreSkippedTxTracesFlag` is set
+	Traces *types.BlockTrace `json:"traces,omitempty"`
```

Comment:
Don't add this here, add it to `RPCTransaction` in `eth/api.go`

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/500/commits/8920cf9daa1f9dac2b48f9e9c485943ebd0ba890